### PR TITLE
New post: How I got started Learning AI-ML

### PR DESCRIPTION
**Pull request by cobusbernard**
# Content Pull Request Checklist

Please check to make sure your content submission meets the following requirements by adding an `x` between the brackets:

- [ ] The title of this pull request reflects the title of my content.
- [ ] I have reviewed the [Content Review Checklist](https://github.com/build-on-aws/content/blob/main/CONTENT_REVIEW_CHECKLIST.md).
- [ ] This pull request contains one piece of content (post or page).

Is this pull request linked to a Content Proposal or Idea Suggestion Issue? If so, please link to it below using the number or full link.

```text
Resolves #5

or

Content for https://github.com/build-on-aws/content/issues/5
```

**By submitting this pull request, I confirm I own this content, have permission to use each image included in this content, and that it will be released under the [CC BY-SA 4.0 SA License](/LICENSE).**

### CODE REVIEW

**Comment by cobusbernard** [comment]
This is the new PR, created from https://github.com/build-on-aws/content/pull/357, please continue review here. Due to making repo public.

**Review by riferrei — APPROVED** [review]
LGTM

File: posts/how-i-got-started-learning-ai-ml/index.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
+---
+title: "How I got started Learning AI-ML?"
```
Comment (damonk10) [inline posts/how-i-got-started-learning-ai-ml/index.md:2]: Arun, I like this piece a lot. I'm leaving this comment up top so it's the first one you see, but it's about the structure of the piece overall. I like how you use Jerry's story as a way into your own AI/ML journey. But I think you finish Jerry's story too early. I would suggest restructuring a little bit:

1. Introduction
2. What is AI/ML (combine this with the Types of AI/ML section that you have later in the piece)
3. How I learned (this is currently your final section, but it's the middle part of your journey into AI/ML)
4. How I used my learning to solve Jerry's problem

File: posts/how-i-got-started-learning-ai-ml/index.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+  - devops
+  - sagemaker
+authorGithubAlias: sarunaaa003
+authorName: Arun Kumar. S
+date: 2023-06-10
+---
+
+## Meet Jerry a community gardener
+
+Jerry is a passionate and new community gardener located in the medium-long time frame US plant hardiness zone 6A growing season and shares the garden produce to his community. Due to the limited seasonal weather it is essential for him to be efficient so he can harvest maximum produce.
```
[the remark this answers is not in the document]
Comment (sarunaaa003) [reply posts/how-i-got-started-learning-ai-ml/index.md:17]: Thanks, I like it

File: posts/how-i-got-started-learning-ai-ml/index.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+Both AI and ML are related but distinct concepts. AI refers to the ability of machines to perform tasks that typically requires human intelligence, such as decision-making, recognizing patterns, and learning from experience. 
+
+ML is a specific technique under AI. It uses data to train a model, and you have a trained model, you're capable of adding more data and predict an outcome based on the initial model. In other words ML uses algorithms and statistical models to allow machines to learn from data without being explicitly programmed.
+
+![AI / ML Hirearchy](images/figure1.jpg "Figure 1. AI / ML Hirearchy")
+
+## Combined Approach to solve Jerry’s problem
+
+I developed a prototype reactive mobile app with reference architecture as below that Jerry uses to submit his plant’s picture. This image is picked by a serverless architecture and goes through multiple stages of iteration. Once the data is ingested a Lambda function processes the image using Amazon Rekognition.
```
[the remark this answers is not in the document]
Comment (sarunaaa003) [reply posts/how-i-got-started-learning-ai-ml/index.md:33]: sure

File: posts/how-i-got-started-learning-ai-ml/index.md (hunk 1 above)
Comment (sarunaaa003) [reply posts/how-i-got-started-learning-ai-ml/index.md:2]: Thanks for the suggestion

**Pull request merged**